docs/backend/openai_api_completions.ipynb

```diff
@@ -214,125 +214,8 @@
-    ")\n",
-    "\n",
-    "response = client.chat.completions.create(\n",
-    "    model=\"meta-llama/Meta-Llama-3.1-8B-Instruct\",\n",
-    "    messages=[\n",
-    "        {\n",
-    "            \"role\": \"user\",\n",
-    "            \"content\": \"Give me the information of the capital of France in the JSON format.\",\n",
-    "        },\n",
-    "    ],\n",
-    "    temperature=0,\n",
-    "    max_tokens=128,\n",
-    "    response_format={\n",
-    "        \"type\": \"json_schema\",\n",
-    "        \"json_schema\": {\"name\": \"foo\", \"schema\": json.loads(json_schema)},\n",
-    "    },\n",
-    ")\n",
-    "\n",
-    "print_highlight(response.choices[0].message.content)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Regular expression (use default \"outlines\" backend)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "response = client.chat.completions.create(\n",
-    "    model=\"meta-llama/Meta-Llama-3.1-8B-Instruct\",\n",
-    "    messages=[\n",
-    "        {\"role\": \"user\", \"content\": \"What is the capital of France?\"},\n",
-    "    ],\n",
-    "    temperature=0,\n",
-    "    max_tokens=128,\n",
-    "    extra_body={\"regex\": \"(Paris|London)\"},\n",
-    ")\n",
-    "\n",
-    "print_highlight(response.choices[0].message.content)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### EBNF (use \"xgrammar\" backend)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# terminate the existing server(that's using default outlines backend) for this demo\n",
-    "terminate_process(server_process)\n",
-    "\n",
-    "# start new server with xgrammar backend\n",
-    "server_process = execute_shell_command(\n",
-    "    \"python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3.1-8B-Instruct --port 30000 --host 0.0.0.0 --grammar-backend xgrammar\"\n",
-    ")\n",
-    "wait_for_server(\"http://localhost:30000\")\n",
-    "\n",
-    "# EBNF example\n",
-    "ebnf_grammar = r\"\"\"\n",
-    "        root ::= \"Hello\" | \"Hi\" | \"Hey\"\n",
-    "        \"\"\"\n",
-    "response = client.chat.completions.create(\n",
-    "    model=\"meta-llama/Meta-Llama-3.1-8B-Instruct\",\n",
-    "    messages=[\n",
-    "        {\"role\": \"system\", \"content\": \"You are a helpful EBNF test bot.\"},\n",
-    "        {\"role\": \"user\", \"content\": \"Say a greeting.\"},\n",
-    "    ],\n",
-    "    temperature=0,\n",
-    "    max_tokens=32,\n",
-    "    extra_body={\"ebnf\": ebnf_grammar},\n",
-    ")\n",
-    "\n",
-    "print_highlight(response.choices[0].message.content)"
+    "For OpenAI compatible structed outputs API, refer to [Structured Outputs](https://docs.sglang.ai/backend/structured_outputs.html#OpenAI-Compatible-API) for more details.\n"
```

Comment:
Use relative reference, see this https://github.com/sgl-project/sglang/blob/1a820e38a2fcc6d0e0324605bb39baec23d81f8d/docs/start/send_request.ipynb#L10

@zhaochenyang20